cmd/cloud_login.go

```diff
@@ -45,13 +52,14 @@ func getCmdCloudLogin(gs *state.GlobalState) *cobra.Command {
 	loginCloudCommand := &cobra.Command{
 		Use:   cloudLoginCommandName,
 		Short: "Authenticate with Grafana Cloud k6",
-		Long: `Authenticate with Grafana Cloud k6.
+		Long: fmt.Sprintf(`Authenticate with Grafana Cloud k6.
 
 This command will authenticate you with Grafana Cloud k6.
 Once authenticated you can start running tests in the cloud by using the "k6 cloud run"
 command, or by executing a test locally and outputting samples to the cloud using
 the "k6 run -o cloud" command.
-`,
+%s
+`, warning),
```

Comment:
There is a `Deprecated` in `cobra.Command ` by setting it so we can see the deprecated message for both info and usage. However, I am not sure if it's colored, so maybe you way is good to go :shrug:

Reply:
I'm in favor of both. I hope the deprecation is useful for the help command.

Reply:
Yeah, I thought about the `Deprecated` field, as we discussed in the other PR, but in this case I decided to go this way because the command per se it's not deprecated, indeed a new one, added in the upcoming release. So, once we drop the support we can just remove the warning, and keep the command as-is.

Wdyt?

Reply:
>this way because the command per se it's not deprecated

That's true. Fair enough. Then, it sounds fine to me to move as-is.

Reply:
I feel most important is the end goal, notifying the customers that the command is deprecated, and will be removed is reached. Keeping in mind the timeline, I'm happy with the current changes :+1: Resolving the tread from my side.